definition.go

```diff
@@ -1143,6 +1147,18 @@ func NewInputObject(config InputObjectConfig) *InputObject {
 	gt.fields = gt.defineFieldMap()
 	return gt
 }
+func (st *InputObject) AddInputField(fieldName string, fieldConfig *InputObjectField) {
+	if fieldName == "" || fieldConfig == nil {
+		return
+	}
+	st.mu.Lock()
```

Comment:
Need to check (R)Lock this mutex anywhere that st.fields or st.typeConfig.Fields are accessed (e.g. `(*InputObject).Fields()`)

Reply:
Good call. Fixed.